### ISSUE TITLE
Fix for consistent and correctly parsed edxapp logs

### ISSUE THREAD

**Issue by robrap** [edx/edx-arch-experiments#768]
**Notes:**

- Here is a [search for edx-edxapp-lms Python logs](https://app.datadoghq.com/logs?query=status%3Ainfo%20source%3Apython%20%22ERROR%22%20service%3Aedx-edxapp-lms%20&agg_m=count&agg_m_source=base&agg_q=service&agg_q_source=base&agg_t=count&cols=core_host%2Ccore_service&fromUser=true&messageDisplay=inline&refresh_mode=sliding&storage=hot&stream_sort=desc&top_n=100&top_o=top&viz=stream&x_missing=true&from_ts=1716322160560&to_ts=1716408560560&live=true) showing `status:info`, but contain "ERROR" in the message. These are not parsing correctly.
- The `edx-edxapp-lms` logs have a variety of formats. Are they all from the same source? Can they be more consistent?
- What format do we want? Do we want consistency with other k8s services? Do we think `[INFO]` is better than `INFO` for searching or configuring in DD? See https://github.com/edx/edx-arch-experiments/issues/664#issuecomment-2256080620 for additional discussion.
- See https://github.com/edx/edx-arch-experiments/issues/664 for a more general version of this ticket for the k8s services. 
  - I don't know if any of this applies once we move to k8s.

**Questions:**

- Relates to the above notes:
  - Do we have incorrect status for any logs?
  - Do we care about inconsistent log formats within the LMS?
  - Do we care about consistency with k8s logs?

**Comment by robrap** [comment]
This will be challenging, as since documented in https://2u-internal.atlassian.net/wiki/spaces/ENG/pages/1277394947/Challenges+with+edx-platform+log+formatting.

[inform] @timmc-edx: As with another ticket, I will mark this as P5 and move to the containerization work. Once the initial containerization is complete, and potentially the log files have been adjusted, we can determine whether we will P4 or close this ticket.
